src/schematools/loaders.py

```diff
@@ -34,11 +40,20 @@ def get_dataset(self, dataset_id: str, prefetch_related: bool = False) -> Datase
         """Gets a dataset from the filesystem for dataset_id."""
         from schematools.utils import dataset_schema_from_id_and_schemas_path
 
-        return dataset_schema_from_id_and_schemas_path(
-            dataset_id,
-            schemas_path=self.schema_url,
-            prefetch_related=prefetch_related,
-        )
+        try:
+            return dataset_schema_from_id_and_schemas_path(
+                dataset_id,
+                schemas_path=self.schema_url,
+                prefetch_related=prefetch_related,
+            )
+        except FileNotFoundError:
+            raise DatasetNotFound(f"Dataset `{dataset_id}` not found.")
```

Comment:
```suggestion
        except FileNotFoundError as e:
            raise DatasetNotFound(f"Dataset `{dataset_id}` not found.") from e
```

Ik meen dat hiervoor ook een flake test draait (in DSO-API wel), maar die zal wel verdrongen worden door de reeks andere fouten.

Reply:
Oh bugger, ik had de `mypy` en `flake8` check nog disabled :-(
Ff aanzetten en fixen.

Reply:
😄